Makefile

```diff
@@ -1,21 +1,18 @@
-BIN := venv/bin
-PYTHON := $(BIN)/python
 DEV_LISTEN_IP := 0.0.0.0
 DEV_PORT := 3000
 
 .PHONY: dev
-dev: venv
-	@echo -e "\e[1m\e[93mRunning on http://$(shell hostname -f):$(DEV_PORT)/\e[0m"
-	$(PYTHON) ./manage.py runserver $(DEV_LISTEN_IP):$(DEV_PORT)
+dev: venv serve
 
-venv: requirements.txt requirements-dev.txt
-	python ./vendor/venv-update venv= venv -ppython3 install= \
-		-r requirements.txt -r requirements-dev.txt
+venv: Pipfile Pipfile.lock
+	pipenv install --dev
```

Comment:
It appears that [`--two` and `--three` are destructive](https://github.com/pypa/pipenv/issues/349), which is a pain, since it means it will destroy a created virtualenv and recreate it each time, so that's not great. I came up with adding this line before the existing on instead, which is a bit messy, but seems to do the trick to create a Python 3 virtualenv if one doesn't exist already without reinstalling it every time:

    pipenv --venv || pipenv install --three --dev

It does have the amusing side-effect that it will install it twice the first time, but the second one will be very fast and subsequent times it will not install twice.

Reply:
Works on my machine; however, it didn't seem to install twice like you said.

Reply:
Actually, maybe we can get away with the [requires] python_version="3" instead of this hack?
Will try tomorrow with the vagrant ubuntu (as my home arch linux defaults to python3 anyways).

Reply:
@jvperrin Changing python_version to "3.4" appears to work on the OCF computers (where 2.7 is default).